cypress/integration/history.ts

```diff
@@ -51,7 +51,7 @@ describe('Test History', () => {
       .should('have.length', 1);
 
     // TODO currently I do not know why but without the wait. test gets flaky
-    //      and fails randomly because cypress is not able to drag the card into the container 
+    //      and fails randomly because cypress is not able to drag the card into the container
```

Comment:
🤷

Reply:
Yeah, Cypress tests can be a little flaky at the moment since there really isn't a proper/native way of handling events (especially dnd events) yet and the best we can do for now is to emulate Craft's dnd behaviour to perform tests.

cc: @ankri

Reply:
Oh you were fixing the linting issue, sorry I thought you were asking about the flaky test comment 😆